.azure-pipelines/pull_request.yml

```diff
@@ -14,10 +14,10 @@ variables:
   target_branch: $[variables['System.PullRequest.TargetBranch']]
 
 stages:
-  - stage: 'notebook_tests'
-    displayName: 'notebook tests'
-    jobs:
-      - template: test_notebooks.yml
+  # - stage: 'notebook_tests'
+  #   displayName: 'notebook tests'
+  #   jobs:
+  #     - template: test_notebooks.yml
```

Comment:
disabled?

Reply:
yes, this is being deleted in the next PR anyway: https://github.com/scipp/ess-notebooks/pull/42

Reply:
I was causing seg faults on the builds